Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Process_attempts_to_kill_a_known_security_or_AV_tool.yml

```diff
@@ -0,0 +1,992 @@
+          ignorecase: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 420,
+          "y": 467
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "34":
+    id: "34"
+    taskid: 0f685d2d-7074-493c-8133-9fd4ca42ea31
+    type: title
+    task:
+      id: 0f685d2d-7074-493c-8133-9fd4ca42ea31
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -78,
+          "y": 2355
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "37":
+    id: "37"
+    taskid: eb992417-3881-4b4e-81fc-d5ac1224bc89
+    type: regular
+    task:
+      id: eb992417-3881-4b4e-81fc-d5ac1224bc89
+      version: -1
+      name: Searches for related alerts in the case
+      description: |
+        This task searches for suspicious alerts related to the incident, with a severity of Medium or higher, to identify any additional indications of malicious activity on the endpoint.
+      scriptName: SearchAlertsV2
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "20"
+    scriptarguments:
+      query:
+        complex:
+          root: alert
+          accessor: parentXDRIncident
+          transformers:
+          - operator: Cut
+            args:
+              delimiter:
+                value:
+                  simple: '-'
+              fields:
+                value:
+                  simple: "2"
+          - operator: concat
+            args:
+              prefix:
+                value:
+                  simple: '((severity:"MEDIUM" or severity:"HIGH" or severity:"CRITICAL") ) and -name:"Process attempts to kill a known security/AV tool" and caseid:'
+              suffix: {}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 761,
+          "y": 600
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "38":
+    id: "38"
+    taskid: 3b214e10-667b-4e2a-8ec4-edc30db2176a
+    type: regular
+    task:
+      id: 3b214e10-667b-4e2a-8ec4-edc30db2176a
+      version: -1
+      name: Terminate Causality (CGO)
+      description: Terminate a process tree by its causality ID.
+      script: '|||core-terminate-causality'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#error#':
+      - "39"
+      '#none#':
+      - "24"
+    scriptarguments:
```

Comment:
We are not adding the incident_id to this action?
<img width="587" height="698" alt="image" src="https://github.com/user-attachments/assets/438dc1e1-42d9-4729-9215-e4d79adc0e9f" />

Reply:
Done